src/chunk.c

```diff
@@ -2182,7 +2186,12 @@ ts_chunk_show_chunks(PG_FUNCTION_ARGS)
 		Cache *hcache;
 		int64 older_than = PG_INT64_MAX;
 		int64 newer_than = PG_INT64_MIN;
+		int64 created_before = PG_INT64_MAX;
+		int64 created_after = PG_INT64_MIN;
 		Oid time_type;
+		Oid arg_type;
+		bool older_newer = false;
```

Comment:
Confused by naming "older newer".... What does it mean?

Reply:
It means that both `older_than` and `newer_than` was specified together.